test/services/carrier_account.test.js

```diff
@@ -56,6 +57,30 @@ describe('CarrierAccount Service', function () {
     }
   });
 
+  it('creates a UPS carrier account', async function () {
+    const accountNumber = '123456789';
+    const type = 'UpsAccount';
+
+    const data = {
+      type: type,
+      account_number: accountNumber,
+    };
+
+    try {
```

Comment:
question: why do we have a try/catch here? This can obfuscate an actual problem in our test suite and should not be introduced. Let's just follow the other libs in how we test without getting too creative here. If we can't create and then immediately delete this account in our tests, we need to know and not bury it.